charts/internal/calico/templates/multus/daemonset-multus-cni.yaml

```diff
@@ -0,0 +1,136 @@
+        gardener.cloud/role: system-component
+    spec:
+      nodeSelector:
+        kubernetes.io/os: linux
+      hostNetwork: true
+      hostPID: true
+      tolerations:
+        # Make sure multus gets scheduled on all nodes.
+        - effect: NoSchedule
+          operator: Exists
+        - effect: NoExecute
+          operator: Exists
+      serviceAccountName: multus
+      securityContext:
+        seccompProfile:
+          type: RuntimeDefault
+      # Minimize downtime during a rolling upgrade or deletion; tell Kubernetes to do a "force
+      # deletion": https://kubernetes.io/docs/concepts/workloads/pods/pod/#termination-of-pods.
+      terminationGracePeriodSeconds: 0
+      priorityClassName: system-node-critical
+      initContainers:
+      # This container installs the multus CNI binaries
+      # and CNI network config file on each node.
+      - name: install-multus-cni
+        image: {{ index .Values.images "multus-cni" }}
+        command: ["/install_multus"]
+        args:
+          - "--type"
+          - "thin"
+        resources:
+          requests:
+            cpu: 10m
+            memory: 15Mi
+        volumeMounts:
+          - name: cnibin
+            mountPath: /host/opt/cni/bin
+            mountPropagation: Bidirectional
+        securityContext:
+          privileged: true
+      {{ if .Values.config.multus.installCNIPlugins }}
+      # This container installs the CNI binaries on each node.
+      - name: install-cni-plugins
+        image: {{ index .Values.images "cni-plugins" }}
+        command: ["/bin/sh"]
+        args:
+          - "-c"
+          - "cp /usr/src/cni/bin/* /host/opt/cni/bin/"
+        resources:
+          requests:
+            cpu: 10m
+            memory: 15Mi
+        volumeMounts:
+          - name: cnibin
+            mountPath: /host/opt/cni/bin
+        securityContext:
+          privileged: true
+      {{ end }}
+      containers:
+        # Runs multus container on each Kubernetes node.
+        - name: multus
+          image: {{ index .Values.images "multus-cni" }}
+          command: ["/thin_entrypoint"]
+          args:
+            - --cleanup-config-on-exit
+            - --cni-bin-dir=/host/opt/cni/bin
+            - --cni-conf-dir=/host/etc/cni/net.d
+            - --cni-version=0.3.1
+            - --multus-autoconfig-dir=/host/etc/cni/net.d
+            - --multus-cni-conf-dir=/etc/cni/multus/net.d
+            - --multus-conf-file=auto
+            - --multus-log-file=/var/log/multus.log
+            - --multus-log-level=error
+            - --multus-log-to-stderr=true
+            - --multus-master-cni-file-name=10-calico.conflist
+            - --readiness-indicator-file=/etc/cni/net.d/10-calico.conflist
+          securityContext:
+            privileged: true
+          resources:
+            requests:
+              cpu: "100m"
+              memory: "200Mi"
+            limits:
+              cpu: "100m"
+              memory: "200Mi"
```

Comment:
As far as I can tell, this container does some initial setup, but afterwards it just does some polling every second that the CNI configuration did not change (see https://github.com/k8snetworkplumbingwg/multus-cni/blob/00adf22482945de41ff702258593aaf867a48e2d/cmd/thin_entrypoint/main.go#L627-L663). For that the resource requests seem a little high. WDYT?

In general, we try to stay away from CPU limits, i.e. we removed them almost everywhere in the gardener code base, because they have potential for a lot of issues, but help rarely. Do you think that we really need them here?